include/aspect/material_model/interface.h

```diff
@@ -217,6 +217,16 @@ namespace aspect
                           const bool use_strain_rates = true);
 
       /**
+       * Copy Constructor.
+       */
+      MaterialModelInputs(const MaterialModelInputs<dim> &material);
+
+      /**
+       * Assignment operator.
+       */
+      MaterialModelInputs &operator=(const MaterialModelInputs &material);
```

Comment:
Removing this operator and the constructor would not help, because they are implicitly created when called, and even worse you then get a cryptic error message that you can not assign a DoFAccessor without a particle handler (apparently `active_cell_iterator a = active_cell_iterator()` is not allowed). That is why we created the constructor and operator. Should we instead mark them as deleted in the header file like this?
```
MaterialModelInputs (...) = delete;
... operator= (...) = delete;
```